pkg/daemon/containernetwork/constants.go

```diff
@@ -20,7 +20,7 @@ const (
 	DockerNetnsDir     = "/var/run/docker/netns"
 	ContainerdNetnsDir = "/var/run/netns/"
 
-	ContainerHostLinkSuffix = "_h"
+	ContainerHostLinkPrefix = "h_"
```

Comment:
What if supporting both prefix and suffix?
1. Check both
2. Create prefix

Then the upgrading will become more smooth.

Reply:
Actually, "ContainerHostLinkPrefix" is only used to generate veth pair name, if in a new environment with newest hybridnet version, "ContainerHostLinkSuffix" is not needed anymore. 

While upgrading, "_h" is also checked only one place of all the codes, so we don't need to keep the "ContainerHostLinkSuffix" variable for upgrading.

Reply:
Unlike "ip-pools" annotation, this will not be used directly by users.